Coder Desktop/Coder Desktop/Session.swift

```diff
@@ -8,9 +9,12 @@ protocol Session: ObservableObject {
 
     func store(baseAccessURL: URL, sessionToken: String)
     func clear()
+    func tunnelProviderProtocol() -> NETunnelProviderProtocol?
 }
 
-class SecureSession: ObservableObject {
+class SecureSession: ObservableObject & Session {
```

Comment:
nit for consistency:
```suggestion
class SecureSession: ObservableObject, Session {
```